src/Avalonia.Base/AvaloniaPropertyMetadata.cs

```diff
@@ -31,6 +33,15 @@ public BindingMode DefaultBindingMode
             }
         }
 
+        public UpdateSourceTrigger UpdateSourceTrigger
```

Comment:
In WPF this is called [`DefaultUpdateSourceTrigger`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkpropertymetadata.defaultupdatesourcetrigger?view=windowsdesktop-6.0) which is maybe a better name